iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -97,8 +97,10 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
         let firstUnread: Message? = self.dataSource.itemIdentifiers(in: .messages)
             .compactMap({ type in
             switch type {
-            case .message(cid: let cid, messageID: let messageID, _):
-                return ChatClient.shared.message(cid: cid, id: messageID)
```

Comment:
This message function needs to be used with care. Every time it's call it triggers a GetMessage call on stream. This was one area where we were hammering the GetMessage endpoint.